.circleci/config.yml

```diff
@@ -259,6 +259,13 @@ jobs:
             echo "Running mvn with SDK version: '$SDK_VERSION'"
             mvn -Dakkaserverless-sdk.version=$SDK_VERSION test-compile
             mvn -Dakkaserverless-sdk.version=$SDK_VERSION verify -Pit
+      - run:
+          name: Event Sourced "Counter" sample
+          command: |
+            cd samples/java-eventsourced-counter
```

Comment:
For the record, I'm prefixing it with `java-` like some (but not all) other samples. 

Since we plan to start work on the Scala SDK, it will be good to prefix the projects with the lang.